UploadHelper/Program.cs

```diff
@@ -0,0 +1,221 @@
+
+    private const string VersionPrefixStart = "<VersionPrefix>";
+    private const string VersionPrefixEnd = "</VersionPrefix>";
+    private const string VersionSuffixStart = "<VersionSuffix>";
+    private const string VersionSuffixEnd = "</VersionSuffix>";
+
+    private static string[] _uploadPageURLs = new string[]
+    {
+        "https://github.com/SubnauticaModding/Nautilus/releases",
+        "https://www.submodica.xyz/mods/sn1/246",
+        "https://www.submodica.xyz/mods/sbz/247",
+        "https://www.nexusmods.com/subnautica/mods/1262",
+        "https://www.nexusmods.com/subnauticabelowzero/mods/373"
+    };
+
+    public static void Main(string[] args)
+    {
+        // essential variables
+        _nautilusDirectory = Path.Combine(Path.GetDirectoryName(Assembly.GetExecutingAssembly().Location), "..", "..", "..");
+
+        // greeting
+        Console.WriteLine("Hello, I’m Lee23 from the Subnautica Modding Organization. So I hear you want to upload a new version of Nautilus?");
+        Console.WriteLine("Please make sure you have a backup of the project or at least some form of source control; this program WILL modify your Nautilus files!\n");
+
+        // explanation
+        Console.WriteLine("I’ll be your personal assistant to speed up the update process for Nautilus. I’ll make sure you remember to follow every step.\n");
+
+        Console.WriteLine("Press ENTER to begin...");
+        Console.ReadLine();
+
+        // get old version
+        Console.WriteLine("First of all, we need to determine the version string.");
+        var oldVersion = GetCurrentVersionString();
+        Console.WriteLine($"I think the current version string is {oldVersion} (suffixes such as the pre-release number might not be shown here), but it's a good idea to double check.\n");
+        Console.WriteLine("When you press enter, I will open https://www.submodica.xyz/mods/sn1/246 and https://github.com/SubnauticaModding/Nautilus/releases in your browser so you can find the version string that is currently uploaded to the internet.");
+        Console.WriteLine("If you don't want this, just type something before you hit enter.");
+        if (string.IsNullOrEmpty(Console.ReadLine()))
+        {
+            Process.Start("explorer", "https://www.submodica.xyz/mods/sn1/246");
+            Thread.Sleep(1000);
+            Process.Start("explorer", "https://github.com/SubnauticaModding/Nautilus/releases");
+        }
+
+        // determine new version
+
+        Console.WriteLine("Please write the NEW version number here (do NOT include the pre-release suffix): ");
+        var versionPrefix = Console.ReadLine();
+        if (versionPrefix != null && versionPrefix.StartsWith("v"))
+        {
+            Console.WriteLine("Hm, why does it start with a V? Are you sure you meant to do that? If not, type R and we can retry that.");
+            var line = Console.ReadLine();
+            if (line != null && line.ToLower() == "R")
```

Comment:
line.ToLower() == “R” doesn’t seem quite right. Wouldn’t the uppercase R stop this condition from ever occurring?

Reply:
Nice catch I forgot that part of the code even existed